project/json-maker/word-info/urls.txt

```diff
@@ -1,10 +1,11 @@
-https://www.cdc.gov/coronavirus/2019-ncov/your-health/need-to-know.html
+CDC: What You Need to Know,https://www.cdc.gov/coronavirus/2019-ncov/your-health/need-to-know.html
+CDC: Older Adults,https://www.cdc.gov/coronavirus/2019-ncov/need-extra-precautions/older-adults.html
 
-https://www.nasa.gov/audience/forstudents/k-4/stories/nasa-knows/what-is-climate-change-k4.html
-https://climate.nasa.gov/news/3072/direct-observations-confirm-that-humans-are-throwing-earths-energy-budget-off-balance/
+NASA on Climate Change,https://www.nasa.gov/audience/forstudents/k-4/stories/nasa-knows/what-is-climate-change-k4.html
```

Comment:
Is NASA on Climate Change supposed to be structured as NASA on Climate Change: ? Some URL sources end with a comma and others with a colon, and I just want to make sure it's intentional.

Reply:
Comma separates link from source. Colon did not have to be used, that was making the source look a bit fancy